software/messaging/src/main/java/org/apache/brooklyn/entity/zookeeper/ZooKeeperNode.java

```diff
@@ -46,7 +46,7 @@
 
     @SetFromFlag("downloadUrl")
     AttributeSensorAndConfigKey<String, String> DOWNLOAD_URL = ConfigKeys.newSensorAndConfigKeyWithDefault(SoftwareProcess.DOWNLOAD_URL,
-            "http://apache.fastbull.org/zookeeper/zookeeper-${version}/zookeeper-${version}.tar.gz");
+            "http://apache.org/dist/zookeeper/zookeeper-${version}/zookeeper-${version}.tar.gz");
```

Comment:
Do we not want to use: `www.apache.org/dyn/closer.cgi?filename=zookeeper/zookeeper-3.4.9/zookeeper-3.4.9.tar.gz&action=download` as described [here](http://www.apache.org/dev/release-download-pages#closer)